phial/wrappers.py

```diff
@@ -68,6 +68,84 @@ def __eq__(self, other: object) -> bool:
         return self.__dict__ == other.__dict__
 
 
+class MessageAttachmentField():
+    '''
+    Displays field in message attachment
+
+    Attributes:
+        title(str): The title of the field
+        value(str): The value of the field
+        short(bool): If false, 2 fields will be displayed on the same line
+
+    '''
+    def __init__(self,
+                 title: Optional[str] = None,
```

Comment:
Does the slack API require any of these to be set? Seems a bit odd that `MessageAttachmentField()` would be valid (as everything is set to `None` by default). If the slack API accepts this without any errors then its fine by me, but should definitely be tested.

Reply:
Just tested this and the Slack API does accept the request but just does not display the attachment

Reply:
Fair enough, if they're truly optional no point adding false constraints